jenkins/jenkinsfile

```diff
@@ -0,0 +1,28 @@
+pipeline {
+    agent {
+         docker {
+            label 'AL2-X64'
+            image 'opensearchstaging/ci-runner:ci-runner-ubuntu1804-build-v1'
```

Comment:
Maybe add link to this docker file: https://github.com/opensearch-project/opensearch-build/blob/main/docker/ci/dockerfiles/build.ubuntu18.opensearch.x64.dockerfile if anyone wants to see whats in the docker image

Reply:
Sounds good. Pushed the change. Thanks for the review :)